### PR TITLE
Create autoyast installation for sle15 sp3 sdk

### PR DESCRIPTION
**Pull request by JRivrain**
See https://progress.opensuse.org/issues/124922

- Added autoyast profile 
- updated subroutine for deregistration, removed outdated workaround and
  added one.
- increased timeout in autoyast installation for s390, as it takes more
  time to start all the services we activate in the new autoyast
profile.
VR https://openqa.suse.de/tests/11360003
children jobs https://openqa.suse.de/tests/11361616 https://openqa.suse.de/tests/11361617
MR
https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/548
https://gitlab.suse.de/coolgw/wegao-test/-/merge_requests/303

### CODE REVIEW

File: data/autoyast_sle15/create_hdd/create_hdd_gnome_sdk_allpatterns_s390x.xml [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+    <reg_server>{{SCC_URL}}</reg_server>
+    <slp_discovery t="boolean">false</slp_discovery>
+  </suse_register>
+  <tftp-server t="map">
+    <start_tftpd t="boolean">false</start_tftpd>
+  </tftp-server>
+  <timezone t="map">
+    <timezone>America/New_York</timezone>
+  </timezone>
+  <user_defaults t="map">
```
Comment (jknphy) [inline data/autoyast_sle15/create_hdd/create_hdd_gnome_sdk_allpatterns_s390x.xml:402]: was created this profile with the --compact option when cloning? looks pretty long apparently.

**Comment by jknphy** [comment]
verification shows failed installation.

**Comment by JRivrain** [comment]
> verification shows failed installation.

Updated

File: tests/autoyast/installation.pm [hunk 2]
```diff
@@ -322,7 +322,7 @@ sub run {
 
     # Cannot verify second stage properly on s390x, so reconnect to already installed system
     if (is_s390x) {
-        reconnect_mgmt_console(timeout => 700, grub_timeout => 180);
+        reconnect_mgmt_console(timeout => 1400, grub_timeout => 360);
```
Comment (jknphy) [inline tests/autoyast/installation.pm:325]: we shouldn't increase the timeout in general, for x86_64 we might hide a bug, could you please add a condition here in another PR or file a ticket for it?

**Pull request merged**